starknet_py/net/models/transaction.py

```diff
@@ -72,7 +72,9 @@ class Declare(AccountTransaction):
     class.
     """
 
-    contract_class: ContractClass
+    contract_class: ContractClass = field(
+        metadata={"marshmallow_field": fields.Nested(ContractClassSchema())}
```

Comment:
Are you sure `fields.Nested` is required here? And if so please tell me why.

Reply:
What alternative do you see here?

Reply:
Just `ContractClassSchema()`?

Reply:
Well it has to be a `dataclasses.field`, doesn't it? It wouldn't work with just
```
contract_class: ContractClass = ContractClassSchema()
```
:/

Reply:
Ok, you're right